pkg/render/guardian.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 Tigera, Inc. All rights reserved.
```

Comment:
Does guardian have defaults for the certificate location and service name hard coded?

Reply:
Yes, they are `https://tigera-packetcapture.tigera-packetcapture.svc.cluster.local` and `/certs/packetcapture/tls.crt`